src/api/app/helpers/webui/package_helper.rb

```diff
@@ -72,4 +72,13 @@ def nbsp(text)
     # But at this point we know it truly is html safe
     result.html_safe
   end
+
+  def humanize_time(secs)
```

Comment:
Why not using:

``fuzzy_time`` or ``fuzyy_time_string`` app/helpers/webui/webui_helper.rb